src/instruments/src/EFB/Settings/Settings.tsx

```diff
@@ -16,386 +18,448 @@ type AdirsButton = {
-                    <div className="flex flex-row">
-                        <SimpleInput
-                            className="w-30 ml-1.5 px-5 py-1.5 text-lg text-gray-300 rounded-lg bg-navy-light
-                            border-2 border-navy-light focus-within:outline-none focus-within:border-teal-light-contrast text-center"
-                            placeholder={accelerationOutAlt}
-                            noLabel
-                            value={accelerationOutAltSetting}
-                            onChange={(event) => handleSetAccelerationOutAlt(event)}
-                        />
+            </div>
+        </div>
+    );
+};
+
+const AircraftConfigurationPage = () => {
+    const [weightUnit, setWeightUnit] = usePersistentProperty('CONFIG_USING_METRIC_UNIT', '1');
+    const [paxSigns, setPaxSigns] = usePersistentProperty('CONFIG_USING_PORTABLE_DEVICES', '0');
+
+    const paxSignsButtons: ButtonType[] = [
+        { name: 'No Smoking', setting: '0' },
+        { name: 'No Portable Device', setting: '1' },
+    ];
+
+    const weightUnitButtons: ButtonType[] = [
+        { name: 'Kg', setting: '1' },
+        { name: 'lbs', setting: '0' },
+    ];
+
+    return (
+        <div className="bg-navy-lighter rounded-xl px-6 shadow-lg divide-y divide-gray-700 flex flex-col">
+            <div className="py-4 flex-grow flex flex-row justify-between items-center">
+                <span className="text-lg text-gray-300 mr-1">Weight Unit</span>
+                <SelectGroup>
+                    {weightUnitButtons.map((button) => (
+                        <SelectItem
+                            onSelect={() => setWeightUnit(button.setting)}
+                            selected={weightUnit === button.setting}
+                        >
+                            {button.name}
+                        </SelectItem>
+                    ))}
+                </SelectGroup>
+            </div>
+
+            <div className="py-4 flex flex-row justify-between items-center">
+                <span className="text-lg text-gray-300">PAX Signs</span>
+                <SelectGroup>
+                    {paxSignsButtons.map((button) => (
+                        <SelectItem
+                            onSelect={() => setPaxSigns(button.setting)}
+                            selected={paxSigns === button.setting}
+                        >
+                            {button.name}
+                        </SelectItem>
+                    ))}
+                </SelectGroup>
+            </div>
+        </div>
+    );
+};
+
+const SimOptionsPage = () => {
+    const [showThrottleSettings, setShowThrottleSettings] = useState(false);
+    const { setShowNavbar } = useContext(SettingsNavbarContext);
+
+    const [adirsAlignTime, setAdirsAlignTime] = usePersistentProperty('CONFIG_ALIGN_TIME', 'REAL');
+    const [_, setAdirsAlignTimeSimVar] = useSimVar('L:A32NX_CONFIG_ADIRS_IR_ALIGN_TIME', 'Enum', Number.MAX_SAFE_INTEGER);
```

Comment:
The convention here for useSimVar is ``[, setSimVar]`` not ``[_, setSimVar]``. I know it was like this in the original code too but may as well correct this.

Reply:
thanks for letting me know : )